### PR TITLE
build multi card page

### PR DESCRIPTION
**Pull request by echappen**
## Changes proposed in this pull request:

- adds category page—example: `/methods/discover/`

## security considerations

None

### CODE REVIEW

File: content/methods/discover/contextual-inquiry.md [hunk 1]
```diff
@@ -1,10 +1,20 @@
 ---
 title: Contextual Inquiry
 permalink: /methods/discover/contextual-inquiry/
+redirect_from:
+  - /contextual-inquiry/
```
[the remark this answers is not in the document]
Comment (echappen) [reply content/methods/discover/contextual-inquiry.md:5]: This redirect is ported from the original site. I've found it's good to retain old `redirect_from`s in order to not get 404's when testing off of the old sitemaps.

[the remark this answers is not in the document]
Comment (echappen) [reply content/methods/discover/contextual-inquiry.md:5]: But I should remove the forward slash

File: content/methods/discover/contextual-inquiry.md [hunk 2]
```diff
@@ -1,10 +1,20 @@
...
 layout: layouts/page
 tags: methods
 eleventyNavigation:
   key: methods
   parent: Discover
   title: Contextual Inquiry
+method:
+  title: Contextual Inquiry
+  what: The product team unobtrusively observes participants at work, with their permission, then asks questions.
+  why: To learn how and why users do what they do; to discover needs and attitudes that might not emerge in an <a href="https://guides.18f.gov/methods/discover/stakeholder-and-user-interviews/" class="usa-link">interview</a> to map how tools, digital and otherwise, interact during complex activities.
```
Comment (jduss4) [inline content/methods/discover/contextual-inquiry.md:15]: [comment] There's some inconsistency around `class="usa-link"`, which appears here but which was removed from the contextual inquiry method.  Was this intentional?

Comment (echappen) [reply content/methods/discover/contextual-inquiry.md:15]: Good catch. I couldn't figure out how to get a dynamic url for the hyperlinks in the markdown using html, so I converted them to markdown. But I can't add a class to a hyperlink in markdown unless it's written in html. It looks like the styling doesn't change when the class is removed, so I'm wondering if we still need a `usa-link` class.

**Review by jduss4 — APPROVED** [review]

File: content/methods/discover/contextual-inquiry.md (hunk 2 above)
Comment (jduss4) [reply content/methods/discover/contextual-inquiry.md:15]: Yeah, it looked the same to me but I didn't take the time to compare the classes.  I vote not to worry about it for right now!

**Pull request merged**